bundles/hash.rst

```diff
@@ -0,0 +1,44 @@
+HashBundle
+==========
+
+The SuluHashBundle adds an easy to use solution for protecting entities and
+documents from being overridden by accident.
+
+Serialization
+-------------
+
+The `HashSerializeEventSubscriber` is the most important part of the
+HashBundle, since it is its job to add the hash of the object to the serialized
+representation being delivered in the response. So the serializer has to be
+used to deliver the response with the hash value automatically.
+
+Hashing
+-------
+
+The `HashSerializeEventSubscriber` uses the `AuditableHasher` for hashing
+documents and entities. Therefore it simply uses the changer (a user) and the
```

Comment:
This hasher calculates a hash based on the user who modified a document and the time when that change happened.